libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -72,14 +72,14 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
 
     // @Param: _DISARMED
     // @DisplayName: Enable logging while disarmed
-    // @Description: If LOG_DISARMED is set to 1 then logging will be enabled while disarmed. This can make for very large logfiles but can help a lot when tracking down startup issues
-    // @Values: 0:Disabled,1:Enabled
+    // @Description: If LOG_DISARMED is set to 1 then logging will be enabled at all times while disarmed. If set to 2 it will be enabled when disarmed, but not if a USB connection is detected, which for example can be used to prevent large unwanted data logs being generated when the the vehicle is connected via USB for log downloads. Logging before arming can make for very large logfiles but can help a lot when tracking down startup issues an dis necessary if logging of EKK replay data is selected via the LOG_REPLAY parameter.
```

Comment:
```suggestion
    // @Description: If LOG_DISARMED is set to 1 then logging will be enabled at all times while disarmed. If set to 2 it will be enabled when disarmed, but not if a USB connection is detected, which for example can be used to prevent large unwanted data logs being generated when the the vehicle is connected via USB for log downloads. Logging before arming can make for very large logfiles but can help a lot when tracking down startup issues and is necessary if logging of EKF replay data is selected via the LOG_REPLAY parameter.
```

Reply:
Done